src/vma/vma_extra.h

```diff
@@ -171,6 +171,128 @@ struct __attribute__ ((packed)) vma_info_t {
 	struct timespec		sw_timestamp;
 };
 
+enum vma_completion_mp_mask {
+	VMA_MP_MASK_HDR_PTR = (1 << 0),
+	VMA_MP_MASK_TIMESTAMP = (1 << 1),
+};
+
+/**
+ * @param comp_mask attributes you want to get see @ref vma_completion_mp_mask
+ * @param payload_ptr pointer to user data no including user header
+ * @param payload_length size of payload_ptr
+ * @param packets  how many packets arrived
+ * @param headers_ptr points to the user header defined when creating the ring
+ * @param headers_ptr_length headers_ptr length
+ * @param hw_timestamp the HW time stamp of the first packet arrived
+ */
```

Comment:
I suggest the following comment:

```
/**
 * @param comp_mask attributes you want to get from vma_cyclic_buffer_read (). see @ref vma_completion_mp_mask
 * @param payload_ptr pointer to user data not including user header
 * @param payload_length size of payload_ptr
 * @param packets how many packets arrived
 * @param headers_ptr points to the user header section within the payload defined when creating the ring
 * @param headers_ptr_length headers_ptr length
 * @param hw_timestamp the HW time stamp of the first packet arrived within the batch
 */
```
#### Introduced in:
`add cyclic function api and enalbe related functions when not using VMA_POLL`